signer/src/storage/postgres.rs

```diff
@@ -1452,10 +1452,17 @@ impl super::DbRead for PgStore {
         .map_err(Error::SqlxQuery)
     }
 
-    async fn get_encrypted_dkg_shares(
+    async fn get_encrypted_dkg_shares<X>(
         &self,
-        aggregate_key: &PublicKey,
-    ) -> Result<Option<model::EncryptedDkgShares>, Error> {
+        aggregate_key: X,
+    ) -> Result<Option<model::EncryptedDkgShares>, Error>
+    where
+        X: Into<PublicKeyXOnly> + Send,
+    {
+        // The first two characters are for the aggregate key column
```

Comment:
nit?
```suggestion
        // The first two characters in the aggregate key column
```

Reply:
Fixed in https://github.com/stacks-network/sbtc/pull/1115/commits/856d1f4273c79d9beb6b3eb96f8c13752f6c3f30 and https://github.com/stacks-network/sbtc/pull/1115/commits/86642684946fb78d68643d7417010881dcdfdf3d